types/lscache/index.d.ts

```diff
@@ -1,6 +1,7 @@
-// Type definitions for lscache v1.0.7
+// Type definitions for lscache v1.3.0
```

Comment:
Definitely Typed doesn't retain patch versions, so you should just put `1.3` here.